docs/topics/PasswordGenerator.adoc

```diff
@@ -38,9 +37,10 @@ image::passphrase_generator.png[]
 
 2. Select the number of words you want to be included in your passphrase by dragging the
 Word Count slider.
-3. In the Word Separator field, enter a character, word, number, or space that you want to use a separator between the words in your passphrase.
+3. In the Word Separator field, enter a character, word, number, or space that you want to use as a separator between the words in your passphrase.
+4. _(Optional)_ You can choose a word case between lower, upper, and title case options.

```

Comment:
I believe the more common term is "capitalized" but we already merged "title case" for the program itself. Looking up "title case" is the specific style of capitalization but IIRC in word processing it's always been just capitalization. Splitting hairs, ignore me.